src/taskProvider.ts

```diff
@@ -107,6 +107,14 @@ export class TaskProvider implements vscode.TaskProvider {
       toolTip: "Run Python functional tests (with Qt display enabled)",
       enabled: true,
     },
+    {
+      group: "Functional Tests",
+      name: "Run tests with option 'golden_display'",
```

Comment:
I would rather have something like `Generate golden tests snapshots` displayed to the user

Reply:
ok